### ISSUE TITLE
Results from automat not in answers?

### ISSUE THREAD

**Issue by cbizon** [ranking-agent/strider#165]
Standup Query 2:
```
{
    "message": {
        "query_graph": {
            "nodes": {
                "n0": {
                    "id": "UniProtKB:P52788",
                    "category": "biolink:Gene"
                },
                "n1": {
                    "category": "biolink:ChemicalSubstance"
                }
            },
            "edges": {
                "e01": {
                    "subject": "n0",
                    "object": "n1"
                }
            }
        }
    }
}
```

This is calling molepro and getting 37 results.

Looking at the logs, it's also calling into automat/pharos (among others) with a query like this:
```
{'nodes': {'n0': {'category': 'biolink:Gene', 'is_set': False, 'id': 'NCBIGene:6611'}, 'n1': {'category': 'biolink:ChemicalSubstance', 'is_set': False}}, 'edges': {'e01': {'subject': 'n1', 'object': 'n0', 'predicate': 'biolink:decreases_activity_of'}}}
```

This returns a result from pharos:
```
[{'node_bindings': {'n0': [{'id': 'NCBIGene:6611'}], 'n1': [{'id': 'CHEMBL.COMPOUND:CHEMBL330673'}]}, 'edge_bindings': {'e01': [{'id': '36cc6bce2fbaf6afb6d68012b1dfe6bf'}]}}, {'node_bindings': {'n0': [{'id': 'NCBIGene:6611'}], 'n1': [{'id': 'CHEMBL.COMPOUND:CHEMBL315995'}]}, 'edge_bindings': {'e01': [{'id': 'bf8b0b59392e1d5e5638739c45f8b476'}]}}]
```

But I don't see CHEMBL.COMPOUND:CHEMBL315995 (or it's preferred id PUBCHEM.COMPOUND:100766) in the strider results.

**Comment by patrickkwang** [comment]
May be another symptom of #169, fixed by #166. Please confirm.

**Comment by cbizon** [comment]
We are now getting results from both molpro and many automats.
